docker-build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-DOCKER_IMG_NAME=phoenixrtos/build
+DOCKER_IMG_NAME=phoenixrtos/build:beta
```

Comment:
As I mentioned earlier - that's not needed as you can see the target you want to run is already being built.